tests/integration/render.int.js

```diff
@@ -67,6 +67,7 @@ describe('Full dummy app render', () => {
 	const random32 = 'asdfasdfasdfasdfasdfasdfasdfasdf';
 	const mockConfig = () => Promise.resolve({
 		API_HOST: 'www.api.meetup.com',
+		API_TIMEOUT: 10,
```

Comment:
probably - the `mockConfig` is now used in enough places that it could be broken out into a `platformMocks` module or something - I'll do that and fix up merge conflicts